recipes/lb-nightly-functions/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "lb-nightly-functions" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lb-nightly-functions-{{ version }}.tar.gz
+  sha256: 476542a0b3ebd7b1b8a81aa9b77ca8802a879594a1f70d361bd18c788faa8943
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - lb-wrapcmd = lb.nightly.functions.common:lb_wrapcmd
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry-core >=1.0.0
+    - python >=3.6,<4.0
+  run:
+    - aiohttp >=3.7.4,<4.0.0
+    - archspec >=0.1.2,<0.2.0
+    - elasticsearch <7.14
+    - gitpython >=3.1.18,<4.0.0
+    - lb-nightly-configuration >=0.1.4,<0.2.0
+    - lb-nightly-db >=0.1,<0.2
+    - lb-nightly-utils >=0.1.2,<0.2.0
+    - lbplatformutils >=4.3.8,<5.0.0
+    - python >=3.6,<4.0
+    - python-gitlab >=2.9.0,<3.0.0
+    - spython >=0.1.14,<0.2.0
+
+test:
+  imports:
+    - lb
+    - lb.nightly.functions
+  commands:
+    - pip check
+    - lb-wrapcmd --help
+  requires:
+    - git
```

Comment:
Should this be a full runtime dependency?

Reply:
I'm not sure, but it fixed [this](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=409235&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=3417) (inspired by https://github.com/conda-forge/staged-recipes/pull/10958#issuecomment-625625359).

Reply:
Ah yes, `gitpython` is intentionally packaged to allow it to use the a non-conda-forge `git` binary.

(When deploying this you should probably add `git` to the `environment.yaml` spec you use.)